HOOKS.md

```diff
@@ -4,20 +4,34 @@ A hook is an executable file that Shell-operator runs when some event occurs. It
 
 The hook receives the data and returns the result via files. Paths to files are passed to the hook via environment variables.
 
-## Initialization: search for and configuration of hooks
+## Shell-operator lifecycle
 
 At startup Shell-operator initializes the hooks:
 
 - The recursive search for hook files is performed in the working directory. You can specify it with `--working-dir` command-line argument or with the `SHELL_OPERATOR_WORKING_DIR` environment variable (the default path is `/hooks`).
-  - The executable files found in the path are considered hooks.
+  - Every executable file found in the path is considered a hook.
 
 - The found hooks are sorted alphabetically according to the directories’ and hooks’ names. Then they are executed with the `--config` flag to get bindings to events in JSON format.
 
-- Shell-operator subscribes to events using received bindings.
+- If hooks configuring is successful, then working queue is filled with `onStartup` hooks.
```

Comment:
If hook's configuration is successful, the workqueue is filled with `onStartup` hooks.